src/server.js

```diff
@@ -1,16 +1,21 @@
-import * as dotenv from 'dotenv';
-dotenv.config();
-
 import express from 'express';
 import cors from 'cors';
+import dotenv from 'dotenv';
 
-import routes from './routes.js';
+import oauthRoute from './routes/oauth.js';
+import studentsRoute from './routes/students.js';
+import usersRoute from './routes/users.js';
+
+dotenv.config();
```

Comment:
A importação do dotenv e o dotenv.config() devem ser feitos no topo do arquivo.

```suggestion
import dotenv from 'dotenv';
dotenv.config();

import express from 'express';
import cors from 'cors';

import oauthRoute from './routes/oauth.js';
import studentsRoute from './routes/students.js';
import usersRoute from './routes/users.js';

```